tests/helpers.py

```diff
@@ -235,9 +258,12 @@ def create_pvc(sc_name, pvc_name=None):
     )
     pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pvc_data['spec']['storageClassName'] = sc_name
-    return create_resource(
-        desired_status=constants.STATUS_BOUND, **pvc_data
-    )
+
+    ocs_obj = pvc.PVC(**pvc_data)
+    pvc_name = pvc_data.get('metadata').get('name')
```

Comment:
no need for this line - simply use ocs_obj.name

Reply:
fixed